docs/datasources/index.mdx

```diff
@@ -16,6 +16,8 @@ Packer supports the following data sources at the moment:
 - [amazon-secretsmanager](/docs/datasources/amazon/secretsmanager) - Retrieve information
 about a Secrets Manager secret version, including its secret value.
 
+- [amazon-parameterstore](/docs/datasources/parameterstore.mdx) - Retrieve information about a parameter in SSM.
```

Comment:
This one can't contain the `.mdx` extension because is rendered on the website
```suggestion
- [amazon-parameterstore](/docs/datasources/parameterstore - Retrieve information about a parameter in SSM.
```